src/plugins/intel_gpu/src/kernel_selector/jitter.cpp

```diff
@@ -1489,6 +1489,39 @@ JitConstants MakeTypeJitConstants(Datatype dataType, const std::string& macroNam
             type_size = "0.5f";
             is_fp = false;
             break;
+        case Datatype::BF16:
+            std::cout << " macroName " << macroName << std::endl;
+            if ( macroName == "INPUT0" || macroName == "INPUT1" || macroName == "INPUT2" || macroName == "INPUT_REORDER" ) {
+                type = "ushort";
+                max_val = "USHORT_MAX";
+                min_val = "0";
+                max_func = "fmax";
+                min_func = "fmin";
+                abs_func = "fabs";
+                type_size = "2";
+                is_fp = false;
```

Comment:
Some of these properties are not really applicable to bf16 type
Let's configure only known properties

Reply:
done